earth_enterprise/rpms/build.gradle

```diff
@@ -0,0 +1,138 @@
+buildscript {
+    repositories {
+        maven {
+            url "https://plugins.gradle.org/m2/"
+        }
+    }
+    dependencies {
+        classpath "com.netflix.nebula:gradle-ospackage-plugin:4.4.2"
+    }
+}
+
+apply plugin: "nebula.ospackage"
+
+def env = System.getenv()
+def home = env['HOME']
+def tmp_install = '/tmp/fusion_os_install'
+def tmp_common = tmp_install + '/common'
+def tmp_common_usermagic = tmp_install + '/common/user_magic'
+def tmp_manual = tmp_install + '/manual'
+def tmp_server_opt = tmp_install + '/server/opt'
+def tmp_server_etc = tmp_install + '/server/etc'
+def tmp_server_apachesupport = tmp_install + '/server/AppacheSupport/'
+def tmp_server_usermagic = tmp_install + '/server/user_magic/'
+
+def tmp_fusion = tmp_install + '/fusion'
+
+// def tmp_searchexample = tmp_install + 'searchexample'
+//geplaces
+
+def base_install_dir = "/test"
+
+apply plugin: 'nebula.ospackage'
+
+ospackage {
+    packageName = 'ge-common'
+    version = '5.2.1'
```

Comment:
We'd want to read the Open GEE version from `earthenterprise/earth_enterprise/src/fusion_version.txt`, but we can also leave that for another ticket.